nohup.out

```diff
@@ -0,0 +1,4 @@
+Nemo-Share-Message: 13:48:21.967: Called "net usershare info" but it failed: 'net usershare' returned error 255: mkdir failed on directory /var/run/samba/msg.lock: Permission denied
+net usershare: cannot open usershare directory /var/lib/samba/usershares. Error No such file or directory
+Please ask your system administrator to enable user sharing.
```

Comment:
Remember what I said last tuesday ? Do not re-read your code in place in the editor, re-read your *diff*. You would not have been able to miss this.